deployment/docker/victorialogs/compose.yml

```diff
@@ -1,26 +1,8 @@
 services:
-  telegraf:
-    image: bitnami/telegraf:1.31.0
-    restart: on-failure
-    volumes:
-      - type: bind
-        source: /var/run/docker.sock
-        target: /var/run/docker.sock
-      - type: bind
-        source: /var/lib/docker
-        target: /var/lib/docker
-      - ./telegraf.conf:/etc/telegraf/telegraf.conf:ro
-    command: --config /etc/telegraf/telegraf.conf
-    depends_on:
-      victorialogs:
-        condition: service_healthy
-      victoriametrics:
-        condition: service_healthy
-
   victorialogs:
     image: docker.io/victoriametrics/victoria-logs:v0.29.0-victorialogs
     volumes:
-      - victorialogs-vector-docker-vl:/vlogs
+      - victorialogs:/vlogs
```

Comment:
this creates conflicts if you run multiple of this

Reply:
it doesn't project name is prepended to each resource name